docs/guides/admin/docs/configuration/security.ldap.md

```diff
@@ -111,8 +110,8 @@ The parameters in this file control the user authorization, i.e. how the roles o
 assigned to the users.
 
 
-Combination with Existing authorization Mechanisms
---------------------------------------------------
+Step 3: Combination with Existing authorization Mechanisms
```

Comment:
This page is no step-by-step guide. This section only explains how things work. It's not a step where people would do something it's a reference for people to check things if they want to know how this works.